homeassistant/components/homeworks/light.py

```diff
@@ -97,11 +97,6 @@ def _set_brightness(self, level: int) -> None:
             float((level * 100.0) / 255.0), self._rate, 0, self._addr
         )
 
-    @property
-    def extra_state_attributes(self) -> dict[str, str]:
-        """Supported attributes."""
-        return {"homeworks_address": self._addr}
-
```

Comment:
Looks like an unrelated change

Reply:
This is moved to the base class so the same code is used both by the light and button entities:
https://github.com/home-assistant/core/blob/9c361765cc88ecf69fda5c6b7b30285d4611023c/homeassistant/components/homeworks/__init__.py#L189

I can drop it from this PR and add it in a separate PR though?

Reply:
No need for that, thnx